devel/mkapidoc.pl

```diff
@@ -42,18 +42,55 @@
     next if m! ^ embed \. fnc \t !x;
     next if m! ^ ( cpan | dist | t) / !x;
     next if m! [^\t]* \.t \t !x;
-    push @files, $_;
+    s/\t.*//;
+    push @files, "$PERLROOT/$_";
 }
+close $m;
+
+# Examine the SEE ALSO section of perlapi which should contain links to all
+# the pods with apidoc entries in them.  Add them to the MANIFEST list.
+my $file;
+
+sub callback {
+    return unless $_ eq $file;
```

Comment:
` unless defined $_ && ....`

Reply:
I would want to see a runtime warning if $_ is undefined.  Is there a reason not to?

Reply:
this works too, this was just a suggestion, I'm fine with whatever